cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -589,26 +589,6 @@
         }
       }
     },
-    "cc_debug": {
```

Comment:
Not sure if it is safe to drop this sub-schema. We could also keep it as deprecated to maintain back-ward compatibility.

Reply:
It's ok to drop this. we shouldn't surface schema for modules that no longer exist. I also don't think there are many use-cases where 'debug' makes sense (and it isn't included in default `/etc/cloud/cloud.cfg` module ordering so folks would have to create custom images with modified /etc/cloud/cloud.cfg.d/ or launch with custom vendor-data to add this module into active boot.